### DISCUSSION TITLE
/cmi rt isn't very random

### DISCUSSION THREAD

**Discussion by Qinak** [Zrips/CMI#2289] [Feature Requests]
**Description of feature/improvement request:**
Improve the random TP `/cmi rt`. Right now you always have 4 close spots you always get teleported to. 
---
### Detailed explanation of what it should do, when and in what situation: ###
A random TP isn't random enough if you have 4 spots you always get teleported close to.
### Basic explanation as to why this feature is needed and/or useful to have: ###
Be more random.

**Comment by mercurialmusic** [comment]
Post your random teleportation config section? What are your values for distances?
